contrib/cirrus/setup_environment.sh

```diff
@@ -95,7 +98,8 @@ esac
 
 # Force the requested database backend without having to use command-line args
 # shellcheck disable=SC2154
-hack/toml-edit "engine.database_backend=$CI_DESIRED_DATABASE"
+printf "[engine]\ndatabase_backend=\"$CI_DESIRED_DATABASE\"\n" > /etc/containers/containers.conf.d/92-db.conf
```

Comment:
It works with the (temporary) `make_safe_containers_conf()` in system tests, which is most of what I care about. Are you suggesting that the `int` test failures in `restart` tests are related to `$CONTAINERS_CONF`? Maybe some other test running concurrently? Seems improbable, because `int` tests (almost) all do explicit command-line options for everything, but we can't rule it out.

Reply:
The problem is that containers.conf.d and other system and user configs are ignored when CONTAINERS_CONF is set.  So if we drop a `containers.conf.d/sqlite.conf`, the tests using CONTAINERS_CONF will ignore this file.

Good news is that https://github.com/containers/common/pull/1378 will address this problem, so I suggest to rather use CONTAINERS_CONF_OVERRIDE in CI here.